firmware/src/network_model.c

```diff
@@ -439,16 +382,13 @@ int NetworkModel_SetStaticAddress(const RDMHeader *header,
     return RDMResponder_BuildNack(header, NR_FORMAT_ERROR);
   }
 
-  unsigned int id = JoinUInt32(param_data[0], param_data[1],
-                               param_data[2], param_data[3]);
-  int index = LookupIndex(id);
+  const int index = LookupIndex(ExtractUInt32(param_data));
   if (index < 0) {
     return RDMResponder_BuildNack(header, NR_DATA_OUT_OF_RANGE);
   }
 
-  uint32_t ip = JoinUInt32(param_data[4], param_data[5],
-                           param_data[6], param_data[7]);
-  uint8_t netmask = param_data[8];
+  const uint32_t ip = ExtractUInt32(&param_data[4]);
```

Comment:
I don't think that improves readability. It's easy for me to check this is doing the right thing by looking at the offsets, rather than following a const.